src/test/groovy/wooga/gradle/release/ReleasePluginSpec.groovy

```diff
@@ -92,6 +93,26 @@ class ReleasePluginSpec extends ProjectSpec {
         "paket-unity"   | PaketUnityPlugin
     }
 
+    def 'Supports adding wdk-unity-release first'() {
```

Comment:
This should work, for this specific case, so I'll approve it, but its very likely that some issues will rise up from mixing `release and `wdk-unity-release` in the same project. Out of my mind, I remember that paket extracts a bunch of stuff to a folder named `packages` which is the same name as the packages folder of the unity project. This may be problematic. 

Please make some RCs to test this and ensure that all of those possible conflicts are sorted out before publishing a final version of this guy :3

Reply:
That was my plan as well.

The whole "packages" clash problematic is handled and was covered in tests in atlas-paket.

Already tested out this whole combo locally, but was meaning to do an RC as well. Is it possible to do it from this branch or does the branch have to be on master?